test_runner/fixtures/neon_fixtures.py

```diff
@@ -3806,6 +3805,19 @@ def create(
         # and make tests more stable.
         config_lines = ["max_replication_write_lag=15MB"] + config_lines
 
+        # Delete file cache if it exists (and we're recreating the endpoint)
+        if (lfc_path := Path(self.lfc_path())).exists():
+            lfc_path.unlink()
+        else:
+            lfc_path.parent.mkdir(parents=True)
+
+        config_lines = [
+            "shared_buffers='512kB'",
+            f"neon.file_cache_path='{self.lfc_path()}'",
+            "neon.max_file_cache_size='512kB'",
+            "neon.file_cache_size_limit='512kB'",
```

Comment:
Apparently this means 0, so let's use 1MB.

```suggestion
            "neon.file_cache_size_limit='1MB'",
```

Reply:
@a-masterov, there's an existing discussion regarding the LFC size for tests, please check [this thread](https://neondb.slack.com/archives/C033A2WE6BZ/p1722942856987979)

Reply:
Thank you, done